### DISCUSSION TITLE
Request Trial Fails

### DISCUSSION THREAD

**Discussion by jelling** [iterate-ch/cyberduck#14979] [Q&A]
- OS: Windows 11
- Install Mountain Duck
- Click "Request Trial"

Get this error:

![image](https://github.com/iterate-ch/cyberduck/assets/1887486/0ed28fb8-7dae-4b8f-809e-324a5c12e5c0)

As you can see in the browser page behind the error, I verified that my machine can reach the `reg.mountainduck.io/trial` URL. But it looks like the `uuid` parameter is empty.

Please advise.

**Comment by dkocher** [comment]
There seems to be a failure retrieving unique identification from your computer. Please request a trial key from [support](mailto:support@mountainduck.io.).
